webconsole/src/main/java/com/pavlovmedia/oss/jaxrs/webconsole/config/JaxrsConsoleConfig.java

```diff
@@ -0,0 +1,10 @@
+package com.pavlovmedia.oss.jaxrs.webconsole.config;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+@ObjectClassDefinition(name="JaxrsConsoleConfig", description="Configuration for JaxrsConsole")
+public @interface JaxrsConsoleConfig {
+    @AttributeDefinition(name="felix.webconsole.label")
+    String felix_webconsole_label() default "JAXRS";
```

Comment:
This variable isn't configurable, it should just be a property passed in.

Reply:
Sure, I will change that.